src/browser/web-view.js

```diff
@@ -327,7 +329,7 @@ define((require, exports, module) => {
       display: 'none'
     },
     passive: {
-      visibility: 'collapsed'
+      visibility: 'collapse'
```

Comment:
Filed a bug to follow up on this https://github.com/mozilla/browser.html/issues/495